### PR TITLE
Add redis conf for CentOS 7 and copy the config

### PR DESCRIPTION
**Pull request by shrivara**
https://github.com/Shippable/admiral/issues/2047

### CODE REVIEW

File: common/scripts/x86_64/CentOS_7/remote/config/redis.conf [hunk 1]
```diff
@@ -0,0 +1,1053 @@
...
+# instance to everybody on the internet. So by default we uncomment the
+# following bind directive, that will force Redis to listen only into
+# the IPv4 lookback interface address (this means Redis will be able to
+# accept connections only from clients running into the same computer it
+# is running).
+#
+# IF YOU ARE SURE YOU WANT YOUR INSTANCE TO LISTEN TO ALL THE INTERFACES
+# JUST COMMENT THE FOLLOWING LINE.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# bind 127.0.0.1
```
Comment (shrivara) [inline common/scripts/x86_64/CentOS_7/remote/config/redis.conf]: From the original file, this was commented out to allow external connections.

File: common/scripts/x86_64/CentOS_7/remote/config/redis.conf [hunk 2]
```diff
@@ -0,0 +1,1053 @@
...
+#
+# The server only accepts connections from clients connecting from the
+# IPv4 and IPv6 loopback addresses 127.0.0.1 and ::1, and from Unix domain
+# sockets.
+#
+# By default protected mode is enabled. You should disable it only if
+# you are sure you want clients from other hosts to connect to Redis
+# even if no authentication is configured, nor a specific set of interfaces
+# are explicitly listed using the "bind" directive.
+protected-mode no
```
Comment (shrivara) [inline common/scripts/x86_64/CentOS_7/remote/config/redis.conf]: This was changed to `no` to allow password-less connection.

File: common/scripts/x86_64/CentOS_7/remote/installRedis.sh [hunk 3]
```diff
@@ -16,6 +16,9 @@ install_redis() {
   sudo yum install -y epel-release
   sudo yum update -y
   sudo yum install -y redis
+  # Move the config that we've copied earlier to the default
+  # location of the config in CentOS.
+  sudo mv /etc/redis/redis.conf /etc/redis.conf
```
Comment (jatindogra) [inline common/scripts/x86_64/CentOS_7/remote/installRedis.sh]: Should we not also do systemctl daemon-reload?
I am not sure whether the changes to conf file will be reloaded or not.

Comment (shrivara) [reply common/scripts/x86_64/CentOS_7/remote/installRedis.sh]: Hmm, didn't seem to need it why I did the install, but I'll have to confirm as I was experimenting.

Comment (shrivara) [reply common/scripts/x86_64/CentOS_7/remote/installRedis.sh]: Actually I think we don't need to to, we're not changing the service definition, but the config that redis loads. So, we should be good.

**Pull request merged**